clickstream/src/main/kotlin/clickstream/internal/eventscheduler/CSEventScheduler.kt

```diff
@@ -115,6 +118,17 @@ internal open class CSEventScheduler(
         val (eventData, eventHealthData) = CSEventData.create(event)
         eventHealthListener.onEventCreated(eventHealthData)
         eventRepository.insertEventData(eventData)
+        dispatchEventToInterceptor {
```

Comment:
Can we add some tests around events being dispatched to interceptor at expected places?